helm/values.yaml

```diff
@@ -2,7 +2,8 @@ environment: development
 postgresUsername: postgres@mine-support2
 postgresPassword: changeme
 postgresDatabase: mine_payments
-postgresHost: ffc-demo-postgres-payments
+postgresHost: ffc-demo-payments-postgres
+postgresServiceName: ffc-demo-payments-postgres
```

Comment:
`postgresHost` is only used internally as the name of the service that proxies communication to the external database.
Both of the values above have to be the same for the service to function, which makes one of them redundant.

The other two database services should be changing to having a single value, rather than changing this one.

Reply:
They need different values in development to support the database running within Kubernetes via `scripts/helm/install` so that we can test the Helm chart offline on a local cluster. I based it on [user service PR#6](https://github.com/DEFRA/ffc-demo-user-service/pull/6).

Reply:
This is adding extra duplicate configuration to the standard release.

Looking at the implementation the development deployment creates an unnecessary service bound to host.docker.internal that isn't used by anything.

A cleaner solution would be to not create the proxy service if the external name is not specified.

That way additional duplicate values are not required in the values file, and the dev deployment doesn't create unnecessary services. Only the postgresHost or an optional postgresExternalName are required.

Reply:
That is a great idea! Thanks.

Reply:
nice one!